Types are out of date with i18next@19.9.0

## 🐛 Bug Report

i18next@19.9.0 includes [type definition changes](https://github.com/i18next/i18next/pull/1560) which cause TypeScript errors when used with i18next-http-backend.

## To Reproduce

```js
import i18next from 'i18next';
import HttpApi from 'i18next-http-backend';

i18next.use(HttpApi).init(i18nextOptions);
```

```
node_modules/i18next-http-backend/index.d.ts:69:3 - error TS2416: Property 'readMulti' in type 'I18NextHttpBackend' is not assignable to the same property in base type 'BackendModule<BackendOptions>'.
  Type '(languages: string[], namespaces: string[], callback: ReadCallback) => void' is not assignable to type '(languages: string[], namespaces: string[], callback: MultiReadCallback) => void'.
    Types of parameters 'callback' and 'callback' are incompatible.
      Types of parameters 'data' and 'data' are incompatible.
        Type 'boolean | ResourceKey | null | undefined' is not assignable to type 'Resource | null | undefined'.
          Type 'string' is not assignable to type 'Resource | null | undefined'.

69   readMulti(languages: string[], namespaces: string[], callback: ReadCallback): void;
     ~~~~~~~~~
```

## Expected behavior

No TypeScript errors when importing i18next-http-backend.

## Your Environment

- *runtime version*: node v14, typescript v4
- *i18next version*: 19.9.0
- *os*: Mac

--- a/index.d.ts
+++ b/index.d.ts
@@ -1,4 +1,4 @@
-import { BackendModule, ReadCallback } from 'i18next';
+import { BackendModule, MultiReadCallback, ReadCallback } from 'i18next';
 
 type LoadPathOption = string | ((lngs: string[], namespaces: string[]) => string);
 
@@ -66,10 +66,10 @@ interface RequestResponse {
 export default class I18NextHttpBackend implements BackendModule<BackendOptions> {
   constructor(services?: any, options?: BackendOptions);
   init(services?: any, options?: BackendOptions): void;
-  readMulti(languages: string[], namespaces: string[], callback: ReadCallback): void;
+  readMulti?(languages: string[], namespaces: string[], callback: MultiReadCallback): void;
   read(language: string, namespace: string, callback: ReadCallback): void;
   loadUrl(url: string, callback: ReadCallback, languages?: string | string[], namespaces?: string | string[]): void;
-  create(languages: string | string[], namespace: string, key: string, fallbackValue: string): void;
+  create?(languages: string | string[], namespace: string, key: string, fallbackValue: string): void;
   type: 'backend';
   services: any;
   options: BackendOptions;

--- a/package.json
+++ b/package.json
@@ -36,7 +36,7 @@
     "eslint-plugin-require-path-exists": "1.1.9",
     "eslint-plugin-standard": "5.0.0",
     "expect.js": "0.3.1",
-    "i18next": "19.8.4",
+    "i18next": "19.9.0",
     "json-server": "0.16.3",
     "json5": "2.1.3",
     "mocha": "8.2.1",

